contracts/smart-wallet/dynamic/DynamicAccount.sol

```diff
@@ -6,6 +6,11 @@ pragma solidity ^0.8.11;
 /* solhint-disable reason-string */
 
 import "../non-upgradeable/Account.sol";
+import "../utils/AccountCore.sol";
+
+import { IERC721Receiver } from "@openzeppelin/contracts/token/ERC721/utils/ERC721Holder.sol";
+import { IERC1155Receiver } from "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";
```

Comment:
do we need these?

Reply:
You're right, this can go in the extension contract instead of the core logic.